skll/utilities/plot_learning_curves.py

```diff
@@ -68,7 +68,7 @@ def main(argv=None):
         makedirs(args.output_dir)
 
     # check that we have pandas and seaborn available
-    if not (_HAVE_PANDAS and _HAVE_SEABORN):
+    if not _HAVE_SEABORN:
         logging.error("Error: need pandas and seaborn to generate learning curve plots.")
```

Comment:
Should we change the error message to remove "pandas and" since that's already installed by default now?

Reply:
Yup. I'll change this. I was being lazy about it because I'm removing `seaborn` anyway in a separate PR, so this whole error message is going away. :)